setup.py

```diff
@@ -39,10 +39,10 @@
         'cartopy>=0.18.0',
         'scikit-learn>=1.0.2',
         'xarray>=0.11.3',
-        #  'seaborn==0.12',
         'emcpy @ git+https://github.com/NOAA-EMC/' +
-        'emcpy@dcdc868e7f48bb1ff55507f34c0548742af8bec1#egg=emcpy',
-        # 'seaborn==0.12',
+        'emcpy@668fab9dcfdd8885cd2cd971478368782cc5db6f#egg=emcpy',
```

Comment:
is this the right hash?

Reply:
Now the PR is merged I've updated the hash to reflect the squash.